graalpy-install-package/src/main/java/com/oracle/example/graalpy/Main.java

```diff
@@ -62,40 +63,77 @@ public static void main(String[] args) throws Exception {
     });
   }
```

Comment:
Maybe print something here at the end of the main, so that the CI log shows at least something :)
```suggestion
    System.out.println("PyfigletFrame initialized");
  }
```